workspace.code-workspace

```diff
@@ -1,20 +1,20 @@
 {
   "folders": [
     {
-      "path": "source\\backend\\api"
+      "path": "source/backend/api"
```

Comment:
I've never seen this file change before - maybe because you were on a Mac?
In any case, this doesn't seem relevant to the pr